### ISSUE TITLE
Support highlight active pane(with border)

### ISSUE THREAD

**Issue by d1y** [zed-industries/zed#8639]
### Check for existing issues

- [X] Completed

### Describe the feature

I am currently looking at @maxbrunsfeld & @maxdeviant, and to be honest, I really like the pink border here. It keeps me focused, and I hope to set this thing in the theme: it can highlight the border of the currently active pane

<img width="1800" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/13c2b04c-2aa8-4e5c-aaf8-a4e4d94a087f">

**Comment by d1y** [comment]
I just tried these two variables, they may be wrong
```json
"experimental.theme_overrides": {
    "panel.focused_border": "#ff0000",
    "pane.focused_border": "#ff0000"
}
```

**Comment by evrsen** [comment]
Just so I understand right, do you want to override the border colour indicating which user you're following with a theme, or do you want themes to be able to highlight the borders of active panes? because if the former, my two cents would be that it's a functionality and not something that should be aesthetically changed by a theme, seeing pink borders when I'm following a green cursor would confuse me. 🤷

**Comment by maxdeviant** [comment]
> I just tried these two variables, they may be wrong
> 
> ```json
> "experimental.theme_overrides": {
>     "panel.focused_border": "#ff0000",
>     "pane.focused_border": "#ff0000"
> }
> ```

The border color is controlled by the player colors. Specifically, I think it's the `cursor` color that we use here.

**Comment by Moshyfawn** [comment]
> do you want to override the border colour indicating which user you're following with a theme, or do you want themes to be able to highlight the borders of active panes? 

I think it's the latter they're looking for as they're looking for a focus pane theme field. So when you focus on a project, editor or terminal pane, they want to see the coloured border around it.

**Comment by evrsen** [comment]
Oh ok in that case I agree, this would be a nice feature to have \:) You've got my upvote

**Comment by d1y** [comment]
I think that's what I want! Can make me more focused cc @zaucy

[modded-zed-ui.webm](https://github.com/zed-industries/zed/assets/45585937/f07dcf7d-1336-4e12-b1a2-ffa8b5893a5c)
